### DISCUSSION TITLE
CTE autocompletion does not work

### DISCUSSION THREAD

**Discussion by ufuk-ergin-carbon** [dbeaver/dbeaver#21007] [Q&A] [answered]
### Description

In this latest release, it is said that "Smart completion for CTE queries (WITH/SELECT) was fixed". However, even a simple CTE autocompletion does not work. Can provide examples if needed. Is this the expected behavior? Here is a screenshot.

<img width="688" alt="Screenshot 2023-08-25 at 11 20 35" src="https://github.com/dbeaver/dbeaver/assets/114302087/9e0b7d51-165b-41fa-be2a-792b869b9f51">

### DBeaver Version

Community Edition Version 23.1.5.202308201919

### Operating System

MacOS Ventura 13.5

### Database and driver

Snowflake Server: 7.29.1
Snowflake Driver: 3.13.6

### Steps to reproduce

1. Make a simple query with easy column aliases.
2. Then wrap it around as a CTE
3. Try selecting columns off of this CTE
4. You'll see that the autocompletion still does not work. Autocompletion for queries not CTE works.

**Comment by E1izabeth** [comment, marked as answer]
It must be a mistake in release notes. We have fixed SELECT with WITH clause in FROM part #20096, but haven't worked on columns autocompletion for selects with CTE and subqueries. Thank you for the interest.

**Comment by ufuk-ergin-carbon** [comment]
Ok thank you. Do you have an estimate as to when you'll introduce autocompletion for CTEs?

**Comment by E1izabeth** [comment]
I can't give any estimates, but we have it in plans.

**Comment by ufuk-ergin-carbon** [comment]
Ok, thanks for the information.
